code/modules/clothing/spacesuits/rig/modules/ninja.dm

```diff
@@ -37,8 +37,8 @@
 
 	var/mob/living/carbon/human/H = holder.wearer
 
-	H << "<font color='blue'><b>You are now invisible to normal detection.</b></font>"
-	H.invisibility = INVISIBILITY_LEVEL_TWO
+	to_chat(H, "<font color='blue'><b>You are now invisible to normal detection.</b></font>")
```

Comment:
Can we change this to "almost invisible", then?

Reply:
It's good enough as-is.

Reply:
It's not *true*, though - 5 alpha isn't full invisibility, it's just really, really good camouflage. If you're moving around in a well-lit area, you're still just barely visible to someone who's paying attention.

Reply:
Yeah, it's really not invisible, that's misleading. I can totally see a ninja running around at alpha 5 if I'm already aware they exist.